src/axes/categoryAxis.ts

```diff
@@ -142,6 +143,12 @@ export class Category extends Axis<string> {
     return this.getDownsampleInfo().domain;
   }
 
+  protected _sizeFromOffer(availableWidth: number, availableHeight: number) {
```

Comment:
what? i don't get why we're doing this here vs just calling super. i see this comment, but if we override this will angled tick labels still request enough space to fit?